types/responses.ts

```diff
@@ -124,6 +159,10 @@ export interface GenerateContentCandidate {
   finishMessage?: string;
   safetyRatings?: SafetyRating[];
   citationMetadata?: CitationMetadata;
+  /** Average log probability score of the candidate. */
+  avgLogProbs?: number;
```

Comment:
A good place to test responses might be in this file: https://github.com/google-gemini/generative-ai-js/blob/main/src/methods/generate-content.test.ts#L86 That's an example of testing citations in the response, which might be the closest thing to this feature. You will need to create a json file that represents a fake response from the backend. In that citation test example, the corresponding file is here: https://github.com/google-gemini/generative-ai-js/blob/0d4fe068cf8deb30ddc14bc3d33446696219bea5/test-utils/mock-responses/unary-success-citations.json You can create it yourself, or if you want to be safe, curl the backend and use a real response. Watch out for carriage returns and hidden whitespace characters if you copy paste from a real response.